src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -225,7 +225,12 @@ internal async Task DeactivateActor(ActorId actorId)
         }
 
         private async Task<T> DispatchInternalAsync<T>(ActorId actorId, ActorMethodContext actorMethodContext, Func<Actor, CancellationToken, Task<T>> actorFunc, CancellationToken cancellationToken)
-        {
+        {            
+            if (!this.activeActors.ContainsKey(actorId))
+            {
+                await this.ActivateActor(actorId);
+            }
+
             if (!this.activeActors.TryGetValue(actorId, out var actor))
```

Comment:
This if can be removed now as the code above will activate it for sure.

Reply:
This call isn't a duplicate - it's calling the one above on line 199.

Reply:
> This call isn't a duplicate - it's calling the one above on line 199.

yes, its fetching the actor instance.  Tt will never go into into the if block. 
You can remove the following line form this if block
"// This should never happen, as "Dapr" runtime activates the actor first. if it ever it would mean a bug in "Dapr" runtime."

Reply:
Oh, it looks like the highlight is pointing at 230-33.  You're pointing at 234.  Ok.

Reply:
> Oh, it looks like the highlight is pointing at 230-33. You're pointing at 234. Ok.

yes, line 234.